challenges/shakespearian-insult-generator.js

```diff
@@ -17,3 +17,11 @@ var second_word = ["weather-bitten", "unchin-snouted", "toad-spotted", "tickle-b
 var third_word = ["wagtail", "whey-face", "vassal", "varlet", "strumpet", "skainsmate", "scut", "ratsbane", "pumpion", "puttock", "pignut", "pigeon-egg", "nut-hook", "mumble-news", "moldwarp", "miscreant", "minnow", "measle", "mammet", "malt-worm", "maggot-pie", "lout", "lewdster", "joithead", "hugger-mugger", "horn-beast", "hedge-pig", "harpy", "haggard", "gudgeon", "giglet", "fustilarian", "foot-licker", "flirt-gill", "flax-wench", "flap-dragon", "dewberry", "death-token", "codpiece", "coxcomb", "clotpole", "clack-dish", "canker-blossom", "bum-bailey", "bugbear", "boar-pig", "bladder", "barnacle", "baggage", "apple-john"];
 
 // YOUR CODE HERE
+function generateInsult()
+{
+  var first = first_word[parseInt(Math.random() * first_word.length)];
+  var second = second_word[parseInt(Math.random() * second_word.length)];
+  var third = third_word[parseInt(Math.random() * third_word.length)];
+  //console.log("You " + first + ", " + second + ", " + third + "");
+  return ("You " + first + ", " + second + ", " + third + "");
```

Comment:
You don't need the last `+ ""` on here--it's "adding" the empty string onto your answer.